File: javascript/fizzbuzz1.js
function fizzbuzz(n){
  // switch statements instead of if...else statements
  const answer=[]
  for (i=1; i<=n; i++){
    switch(true){
      case(i%3===0 && i%5==0):
        answer.push("FizzBuzz")
        break;
      case(i%3==0):
        answer.push("Fizz")
        break;
      case(i%5===0):
        answer.push("Buzz")
        break;
      default:
        answer.push(i.toString())
    }
  }
  return answer
}

console.log(fizzbuzz(15))
// console.log(fizzbuzz(365))

File: lua/fizzbuzz1.lua
-- Learning lua by fizz buzz

print ("Hello, World!")function fizzbuzz(n)
  if n % 15 == 0 then
      return "FizzBuzz"
  elseif n % 3 == 0 then
      return "Fizz"
  elseif n % 5 == 0 then
      return "Buzz"
  else
      return n
  end
end

for i = 1, 100 do
  print(fizzbuzz(i))
end

-- cool

--(@-o) coroutines ... 